ci/convert-e2e-coverage

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env php
+<?php
+
+/**
+ * Converts raw E2E coverage PHP files to PHPUnit .cov format.
+ *
+ * This script loads all the raw xdebug coverage arrays and merges them
+ * into a proper SebastianBergmann\CodeCoverage\CodeCoverage object.
+ *
+ * Usage:
+ *   ./convert-e2e-coverage <input-dir> <output-file>
+ *   ./convert-e2e-coverage <input-dir> <output-file> --clover=<clover-file>
+ *   ./convert-e2e-coverage <input-dir> <output-file> --no-clover
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Michael A. Smith <michael@opencoreemr.com>
+ * @copyright Copyright (c) 2025 OpenCoreEMR Inc.
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
```

Comment:
hi @kojiromike , since this is a php script, should we make sure run from command line only?
(ie. `if (php_sapi_name() !== 'cli') {`)

Reply:
or give it that guardrail :)